state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1180,30 +1192,82 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Associates `short_id` with `affiliate_id` for a given broker. Overwrites the record
-		/// under `short_id` if already taken by another affiliate.
+		/// Registers an affiliate for a broker. Sets the withdrawal address for the affiliate which
+		/// will be used to withdraw the affiliate's fees. The withdrawal address can not be changed
+		/// once set. The broker will be able to trigger a withdrawal
+		/// request to the affiliate's withdrawal address. A short id is taken and can only be used
+		/// at a time. The affiliate account is derived from the broker account id.
```

Comment:
Is this what you intended?
```suggestion
		/// once. The affiliate account is derived from the broker account id.
```